tests/emscripten_main_loop_and_blocker.cpp

```diff
@@ -48,4 +52,5 @@ int main() {
   prevTime = emscripten_get_now();
   emscripten_push_uncounted_main_loop_blocker(main_loop_blocker, NULL);
   emscripten_set_main_loop(looper, 60, 1);
+  return 0;
```

Comment:
this 0 can be confused with the exit(0) above, how about a different code to be sure the right one is hit?